src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Ignore Spelling: Nullable
```

Comment:
What's this?

Reply:
Wacky.  That looks to be something the spell-checking extension did...

I have no idea why it placed it in the source file.  It should be placing in a local dictionary file instead.  Very sorry about that.  Removing.